code/modules/admin/admin.dm

```diff
@@ -728,7 +729,7 @@ var/global/noir = 0
 
 				var/list/valid_modes = list("secret","action","intrigue","random","traitor","meteor","extended","monkey",
 				"nuclear","blob","restructuring","wizard","revolution", "revolution_extended","malfunction",
-				"spy","gang","disaster","changeling","vampire","mixed","mixed_rp", "construction","conspiracy","spy_theft","battle_royale", "vampire","assday", "football")
+				"spy","gang","disaster","changeling","vampire","mixed","mixed_rp", "construction","conspiracy","spy_theft","battle_royale", "vampire","assday", "football", "flock")
```

Comment:
I don't think this is a good idea, especially as most of the game_mode checks are completely unfinished.

Reply:
That only adds it to the list of modes admins can set not as a game mode playable under usual circumstances.

Reply:
Yeah, still. We've given people buttons that say PLEASE DO NOT PRESS THIS, and people still press it vOv.

Reply:
i could add a "Probably wont work well, press at own risk" to it.